hare/algorithm.go

```diff
@@ -498,7 +500,9 @@ func (proc *consensusProcess) processMsg(ctx context.Context, m *Msg) {
 		log.String("msg_type", m.InnerMsg.Type.String()),
 		log.Int("num_values", len(m.InnerMsg.Values)))
 
-	reportLatency(m.InnerMsg.Type, m.Round, proc.clock)
+	// Report the latency since the beginning of the round
+	latency := time.Since(proc.clock.RoundEnd(m.Round - 1))
+	metrics.ReportMessageLatency(pubsub.HareProtocol, strings.ToLower(m.InnerMsg.Type.String()), latency)
```

Comment:
could you change m.InnerMsg.Type.String() to return already lowercased string? ToLower involves string copy and allocation on heap, it is notthing huge, but there is also no reason for doing so